src/utils/errors.ts

```diff
@@ -0,0 +1,63 @@
+import { EthersError, PriceFetchingError, SdkError, SimulationError, TenderlyError, ZapperError } from '@yfi/sdk';
+
+const ETHERS_ERRORS = {
+  [EthersError.FAIL_TOKEN_FETCH]: 'Error fetching token',
+  [EthersError.NO_DECIMALS]: 'No decimals set for vault',
+  [EthersError.NO_PRICE_PER_SHARE]: 'No price per share set for vault',
+  [EthersError.POPULATING_TRANSACTION]: 'Error populating transaction',
+};
+const PRICE_FETCHING_ERRORS = {
+  [PriceFetchingError.FETCHING_PRICE_PICKLE]: 'Error fetching price from pickle',
+  [PriceFetchingError.FETCHING_PRICE_ORACLE]: 'Error fetching price from oracle',
+};
+const SDK_ERRORS = {
+  [SdkError.NO_SLIPPAGE]: 'No slippage set',
+};
+const SIMULATION_ERRORS = {
+  [SimulationError.NO_LOG]: 'No log of transfering token',
```

Comment:
This error is not so clear to me. Can we add some more context or rephrase it?

Reply:
can we translate them? There should be an errors file or object in the translates

Reply:
nvm, doesn't make sense to translate some errors, and have others not translated

Reply:
I mean it does make sense to translate some errors, it's just the start of it, we can start having translation for some and then at some point have translations for them all. I can see how to consume the i18n module from here if it's alright

Reply:
ok, It is just weird to sometimes display errors translated, and sometimes not